packages/fxa-content-server/tests/functional/settings_v2/lib/helpers.js

```diff
@@ -2,19 +2,24 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+const intern = require('intern').default;
 const selectors = require('../../lib/selectors');
 const FunctionalHelpers = require('../../lib/helpers');
 
-const { createEmail } = FunctionalHelpers;
+const { createEmail, generateTotpCode } = FunctionalHelpers;
 const config = intern._config;
 const EMAIL_FIRST = config.fxaContentRoot;
 const SETTINGS_V2_URL = config.fxaSettingsV2Root;
 const password = 'passwordzxcv';
+let secret;
+let securityCode;
```

Comment:
These variable should not be scoped to the module.  They can be in the functions where they are used.